python-test/features/steps/page_objects.py

```diff
@@ -0,0 +1,19 @@
+#XPATHs
```

Comment:
this was a good idea. maybe add a namespace to the variables so we can know that it comes from this file.

like a prefix to know that the variables are coming from this file.  ex: orb_page_*

Or add a class and export variables through that. example OrbPage object and you get `OrbPage.fleet_management_left_menu` and so on.

I think that second approach is more Object Oriented.

Reply:
oh, interesting and good idea.. i'm gonna working on that